introduction_to_applying_machine_learning/synthetic_churn_prediction_with_text/synthetic_churn_prediction_with_text.ipynb

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
<p>Always use present tense -&gt; "we'll" should be "we", "will be used" should be "are used"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>